src/components/atoms/Checkbox.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+interface IProps {
+  className?: string;
+  id: string;
+  setState: React.Dispatch<React.SetStateAction<boolean>>;
+  value: boolean;
+}
+
+const Checkbox: React.FC<IProps> = props => {
+  const updateState = (e: React.FormEvent<HTMLInputElement>) => props.setState(e.currentTarget.checked);
```

Comment:
@FredrikAugust I prefer a different naming convention for event handlers. In stead of passing down a prop called setState, call it onChange, onClick etc.

What are your thinking?